scripts/cli-util/get-team.ts

```diff
@@ -65,7 +65,22 @@ export const handleGetTeam = async () => {
 };
 
 const getSchedule = async (team: TeamData) => {
-	console.log('IN PROGRESS');
+	const schedules: ScheduleData[] | null = await Schedule.find({
+		team: team._id,
+	});
+	schedules.forEach(schedule => {
+		console.log('Schedule:');
+		console.log(`Zoom: ${schedule.zoom}`);
```

Comment:
we no longer need zoom for schedule. ScheduleData model is really outdated. but we do need the table number (can be found in the team's document)